### ISSUE TITLE
Syntactically invalid result when formatting lambda with laziness annotation

### ISSUE THREAD

**Issue by expipiplus1** [lspitzner/brittany#228]
```
$ brittany --output-on-errors
a = \ ~x -> x
ERROR: brittany pretty printer returned syntactically invalid result.
a = \~x -> x
```

Brittany abuts the `~` with `\` which is invalid

**Comment by lspitzner** [comment]
Fixed in 6879436e6721adb3b91f3b32072cd02019b20e8a.

**Comment by expipiplus1** [comment]
Thanks!
